### ISSUE TITLE
Speed Devils lagging geometry

### ISSUE THREAD

**Issue by Senryoku** [Senryoku/Deecy#46]
Weird title.
This is extremely obvious in the Mexico track when moving left to right. Some geometry seems to be "lagging behind", detaching from the rest.
See the tree tops here:
![image](https://github.com/user-attachments/assets/a00aa1c4-0728-4b40-82f0-15945c719ac8)

https://youtu.be/eac4TzM6Wfw

There are a bunch of other visual oddities in this game that I suspect are directly linked to this.

**Comment by Senryoku** [comment]
Rayman 2 has the same issue.

**Comment by Senryoku** [comment]
In Speed Devils the Translucent is literally one frame (or 2 vblanks) old when Start Render is called. Probably a timing issue somewhere, but I have no clue where for now.
```
info(holly):   End of List: OpaqueModifierVolume
info(holly):   End of List: Translucent

(Two VBlanks)

info(holly): TODO SOFTRESET: 00000001
info(holly):   TODO: Tile Accelerator Soft Reset
info(holly): TODO SOFTRESET: 00000000
info(holly):   End of List: Opaque
info(holly): STARTRENDER!
```

**Comment by Senryoku** [comment]
Should be fixed by 7fa811c5356d9ed9138bd8e768705134d32c439e
